conf.d/enhancd.fish

```diff
@@ -1,64 +1,70 @@
-# enhancd-fish initialization hook
-#
-# You can use the following variables in this file:
-# * $package       package name
-# * $path          package path
-# * $dependencies  package dependencies
+function __enhancd_install --on-event enhancd_install
+    # set variables
+    set -Ux ENHANCD_FILTER
```

Comment:
@gazorby Why the universal scope?

Reply:
@jorgebucaran because `__enhancd_install` is called only on plugin install and `ENHANCD_FILTER` is accessed each time `enhancd` is used.

Reply:
@gazorby It's one way, but not the only way. We could also set them outside the event handler since `conf.d` snippets are loaded on startup.

Reply:
@jorgebucaran That was how it was done before fisher 4.0, I'm just trying to leverage fisher event handlers from your latest release. I feel that they are appropriate for this kind of situation, It avoids bloating shell startup and removes the need to check for variables existence (`set -q var; or set -gx var value`), making the snippet a bit more readable.